phoenix-core/src/main/java/org/apache/phoenix/query/QueryConstants.java

```diff
@@ -138,6 +138,9 @@
     public final static String SYSTEM_SCHEMA_NAME = "SYSTEM";
     public final static byte[] SYSTEM_SCHEMA_NAME_BYTES = Bytes.toBytes(SYSTEM_SCHEMA_NAME);
     public final static String PHOENIX_METADATA = "table";
+    public final static String OFFSET = "offset";
+    public final static byte[] offsetRowKeyBytes = "_OFFSET_".getBytes();
```

Comment:
Nit: use Bytes.toBytes(OFFSET) instead. Also, did you want it to be "_OFFSET_" for the OFFSET constant?